Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.4.0"),
-        .package(url: "https://github.com/maiqingqiang/Jinja", from: "1.0.6")
+        .package(url: "https://github.com/johnmai-dev/Jinja", from: "1.1.0")
```

Comment:
I'm going to move this and the chat template test to a new PR, since the rest of the fixes here are more general and unrelated to the new jinja engine.